05_syntax_lab/07.py

```diff
@@ -1,8 +1,35 @@
-""" Write a program that selects a random number
-and asks the user to guess it.
-
-After each guess print a hint "too large" or "too small" to the user.
-
-Bonus: To make things interesting, the program should cheat once in a white
-"""
-
+#commited by Shlomit
+
+from random import randint
+
+def ReadNumsAndCompare():

```

Comment:
Most python keywords and functions are written in lowercase. By your use of functions (which was not yet learned in the course) I assume you already know some python (which is great!). Do take some time to read Python Style Guide (PEP8) and specifically naming conventions there so your code would feel more professional:
https://www.python.org/dev/peps/pep-0008/